Utilities/Thread.cpp

```diff
@@ -2808,3 +2809,12 @@ std::pair<void*, std::size_t> thread_ctrl::get_thread_stack()
 #endif
 	return {saddr, ssize};
 }
+
+template <typename T>
+void lf_queue<T>::wait() noexcept
```

Comment:
I won't be saying "how to use templates" but this is pretty wrong as an idea and repeats past mistakes.
Basically, lf_queue, just like an atomic, is a trivial "leaf" thing and it's not supposed to create circular references like this. If it's told "wait", it should wait, it's not supposed to know anything about threads or other shit.

Reply:
Fixed.